tests/Aspire.Cli.Tests/Utils/CliTestHelper.cs

```diff
@@ -27,13 +28,49 @@ public static IServiceCollection CreateServiceCollection(ITestOutputHelper outpu
 
         var services = new ServiceCollection();
 
-        var configuration = new ConfigurationBuilder().Build();
+        // Build configuration similar to Program.cs but for testing
```

Comment:
The CliTestHelper should not try to find settings files since we will probably want to provide settings files to support each individual test case. The CliServiceCollectionTestOptions should provide two properties (LocalSettingsContent and GlobalSettingsContent) which can optionally be specified.